packages/@secretlint/formatter/src/index.ts

```diff
@@ -85,17 +92,59 @@ export function createFormatter(formatterConfig: SecretLintFormatterConfig) {
      */
     const enableTerminalLink = isHumanReadableFormat ?? formatterConfig.terminalLink ?? false;
     debug(`formatterName: ${formatterName}`);
-    const format = textlintCreateFormatter(formatterConfig);
-    return {
-        format: (results: SecretLintCoreResult[]) => {
-            return format(
-                results.map((result) =>
-                    convertSecretLintResultToTextlintResult(result, {
-                        enableTerminalLink,
-                    })
-                )
-            );
-        },
+
+    try {
```

Comment:
📝 secretlint build-in formatter → textlint formatter as fallback.

Reply:
Sorry for the big delay, some unexpected real life stuff.
Didn't get this comment. You want to document it somewhere?

Reply:
It ok that we do not document it.